lib/improver/utilities/cube_manipulation.py

```diff
@@ -261,6 +261,7 @@ def equalise_cubes(cubes_in, merging=True):
     _strip_var_names(cubes)
     if merging:
         cubelist = _equalise_cube_coords(cubes)
+        cubelist = _equalise_cell_methods(cubelist)
```

Comment:
Does the comment above this section need editing now that the cell_methods are handled to some degree?

Reply:
Comment removed